content/css/concepts/selectors/selectors.md

```diff
@@ -50,7 +50,7 @@ p {
 
 ## Selecting by Attribute
 
-Most commonly, the `class` and `id` attributes are assigned to elements for styling purposes. An `id` cannot start with a number.
+The `class` and `id` attributes are generally assigned for styling different elements on the same page and linking to sections or other pages of the site.
```

Comment:
[Proofreading] 
Hi @Kadif-Ravik! :) Consider using this suggestion as an idea to apply SSwiniarski's suggestions. 
```suggestion
The `class` selector selects elements that are assigned under a specific attribute, a class, given to an HTML element, can be used multiple times, and begins with a period, `.`. Similarly, the `id` selector selects an HTML element that has an `id` attribute. In contrast, the `id` selector is **only** used once and begins with the hashtag symbol, `#`.  
```